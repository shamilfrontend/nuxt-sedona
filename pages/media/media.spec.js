import { shallowMount } from '@vue/test-utils'
import Component from './index'

describe('MediaPage', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(Component, {
      computed: {
        hotels: () => []
      }
    })
  })

  test('mounts properly', () => {
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  test('should match snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
