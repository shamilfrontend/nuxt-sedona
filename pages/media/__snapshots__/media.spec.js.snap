// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MediaPage should match snapshot 1`] = `
<div class="page-wrap">
  <h1 class="page-wrap__title">Фото и видео</h1>
  <div class="page-wrap__content">
    <h2>Фотографии</h2>
    <ul class="photos">
      <li class="photo"><img src="./img/1.jpg" alt="photo" class="photo__img"></li>
      <li class="photo"><img src="./img/2.jpg" alt="photo" class="photo__img"></li>
      <li class="photo"><img src="./img/3.jpg" alt="photo" class="photo__img"></li>
      <li class="photo"><img src="./img/4.jpg" alt="photo" class="photo__img"></li>
      <li class="photo"><img src="./img/5.jpg" alt="photo" class="photo__img"></li>
      <li class="photo"><img src="./img/6.jpg" alt="photo" class="photo__img"></li>
      <li class="photo"><img src="./img/7.jpg" alt="photo" class="photo__img"></li>
      <li class="photo"><img src="./img/3.jpg" alt="photo" class="photo__img"></li>
    </ul>
    <h2>Видео</h2>
    <div class="video-wrap"><iframe src="https://www.youtube.com/embed/gA6pxmdMiOQ" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="allowfullscreen"></iframe></div>
  </div>
</div>
`;
